### ISSUE TITLE
Camera.flyTo rectangle incorrect in 2D 

### ISSUE THREAD

**Issue by hpinkos** [AnalyticalGraphicsInc/cesium#3688]
Reported in #3686

`camear.setView` goes to the rectangle, `camera.flyTo` does not.
``` javascript
var viewer = new Cesium.Viewer('cesiumContainer', {sceneMode: Cesium.SceneMode.SCENE2D});
var scene = viewer.scene;
var clock = viewer.clock;

var west = 0.3323436621771766;
var south = 0.8292930502744068;
var east = 0.3325710961342694;
var north = 0.8297059734014236;
var rectangle = new Cesium.Rectangle(west, south, east, north);
viewer.entities.add({
    rectangle : {
        coordinates : rectangle,
        fill : false,
        outline : true,
        outlineColor : Cesium.Color.WHITE
    }
});

Sandcastle.addToolbarButton('setView', function() {
    viewer.camera.setView({
        destination: rectangle
    });
});

Sandcastle.addToolbarButton('flyTo', function() {
    viewer.camera.flyTo({
        destination: rectangle
    });
});
```

**Comment by novacto2** [comment]
Could you please take a look at it #3695 ? 
@hpinkos

**Comment by hpinkos** [comment]
fixed in #3695
